simplenote_cli/utils.py

```diff
@@ -46,6 +47,12 @@ def get_note_flags(note):
         flags += '   '
     return flags
 
+def get_note_lines(note, max_lines=0):
```

Comment:
@lostways could you remove this optional argument?  It's not used anywhere.

Reply:
Fixed!